.changeset/rare-cherries-bake.md

```diff
@@ -0,0 +1,5 @@
+---
+"@lit/react": patch
```

Comment:
With the ssr react package also being updated, add it to the changeset

Reply:
Done, I think :)

Reply:
Thanks 👍 
I think you can remove the original changeset, as a single changset file should be enough.